addons/website_slides_survey/models/slide_slide.py

```diff
@@ -65,3 +65,28 @@ def _action_set_viewed(self, target_partner, quiz_attempts_inc=False):
                         'slide_partner_id': new_slide_partner.id
                     }
                 )
+
+    def _get_slide_certification_data(self):
```

Comment:
Method name is a little weird.
Since this generates a user_input or uses an existing one and returns the URL, maybe call it `_generate_certification_url`.

Reply:
Done

Reply:
Done